core/src/main/resources/error/error-classes.json

```diff
@@ -101,14 +101,14 @@
     "message" : [ "Field name %s is invalid: %s is not a struct." ],
     "sqlState" : "42000"
   },
+  "INVALID_FORMAT_FOR_CAST" : {
+    "message" : [ "Invalid input value for type %s: %s. To return NULL instead, use 'try_cast'. If necessary set %s to false to bypass this error.%s" ],
```

Comment:
Shall we keep the previous wording
```
Invalid input syntax for type ...
```
It is from PostgreSQL

Reply:
OK